src/hapi/pipelines/database/admins.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         self.admin1_data = {}
         self.admin2_data = {}
 
-    def populate(self):
+    def populate(self) -> None:
```

Comment:
How did you catch all these, are you using a typing extension? Should we add something to pre-commit maybe?

Reply:
I stumbled across one and then noticed it was like that for all of them. It wasn't anything automated. Such a check could help. mypy is one tool used for type checking that probably could be set up in pre-commit.

Reply:
This change is something for a separate PR.